http/swagger.yml

```diff
@@ -2282,7 +2282,17 @@ paths:
           content:
             application/json:
               schema:
-                $ref: "#/components/schemas/Dashboard"
```

Comment:
this old ref is actually inaccurate for this endpoint. The new stuff below represents what the API supports